packages/router/src/router.tsx

```diff
@@ -223,11 +293,7 @@ const RouterImpl = ({
     return <SplashPage isRedwood={isRedwood} />
```

Comment:
The line above here we use `__REDWOOD__`, which TS complains about, because it doesn't know what that is.
How can I fix that? I tried adding it to index.d.ts, but it's not picking it up.

Reply:
I think we intended for the router to be used in other places, but honestly that's not the case anymore. I would remove this flag and just assume that it's always used in a redwood project.